lib/src/models/image/image.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:meta/meta.dart';
+
+import '../../api.dart';
+import '../../get_image_url.dart' as get_image_url;
+import '../../hosts.dart';
+import 'cover.dart';
+import 'image_model.dart';
+import 'image_thumbnail.dart';
+import 'image_type.dart';
+
+
+class Image implements ImageModel {
+  // Work around ImageModelMappable overriding ==.
+  Image(ImageModel image) : _image = image;
+
+  final ImageModel _image;
+
+  @override int get media => _image.media;
+  @override int get id => _image.id;
+  @override ImageType get type => _image.type;
+  @override int? get width => _image.width;
+  @override int? get height => _image.height;
```

Comment:
That is simply bad.

Reply:
Same goes for descendants. If there's some problem with inheriting from classes that have Mappable, open issue in dart_mappable regarding this.